docs/fieldnorm/Software life cycle processes.md

```diff
@@ -0,0 +1,67 @@
+
+  b.  Laboratories will have to document the architecture when the software is classified as class B or C according to IEC-62304.
+
+## 1.2.4 Development
+The development stage will be the bulk of a project; here, code is created that will eventually define the software. The following processes are not required, as smaller bioinformatics teams might find them too difficult to incorporate. Nonetheless, it is strongly recommended that your guidelines include:
+
+  a.  The usage of a decentralized code management system, such as git, together with a centralized repository, such as Github, Gitlab, or Bitbucket, that is seamlessly integrated with a task management system.
+
+  b.  Writing code with unit and integration tests. Moreover, conducting regular code reviews is highly recommended. These methods will be invaluable during software maintenance, testing, and release. It enables a wider understanding of parts of the code, as unit tests often embody general usage as well as very specific exceptions, and code reviews enforce colleagues to have a look and understand what your code is doing.
+
+  c.  Establishing a well-defined development & release cycle with set time periods for preparation, development, testing, & releasing. This allows multiple developers to work cohesively on the project and creates structured testing and release moments during the implementation and maintenance processes. It also enables ample opportunities for user reflection on the deliverables during the lifetime of projects.
+
+## 1.2.5 Testing
+Before implementation, software should be extensively tested, verified, and validated. While ISO 15189 does not provide specific guidelines on how to conduct software validation, it emphasizes the importance of ensuring that software is validated and verified following recognized industry standards. Software validation and verification should be performed in accordance with this document (_link to verification_validation_).
+
+## 1.2.6 Implementation
+Once everything is ready, there are a few key processes that should be included in guidelines to facilitate the release of the software:
+
+  a.  The process for establishing a release. The exact time point at which software is released could be determined by a Development & Release cycle or be released in an independent manner. 
+  
+  b.  The process for performing a release should be well defined. 
+  
+  c.  The software functionality and release date should be clearly communicated to end users. 
+
+## 1.2.7 Maintenance
+Maintenance primarily centers around fixing bugs, monitoring the software, introducing new features, and upgrading to newer versions with new requirements. The process follows the same procedures as the development, testing, and implementation phases. Only certain maintenance processes that could possibly deviate from the normal procedures for development, testing, and implementation should be documented (hotfixes, for example).
```

Comment:
Do we need a section for the hotfix?

Reply:
Yes, we decided that we should add a section off this under "development".

Reply:
I have added a section on hotfixes at the bottom of this subject. it is small but states that there should be something about them somewhere in your guidelines. For now I feel like this is enough, if in the future we ever feel like there should be rules to hotfixes we can always add them later